src/Model/MyParcelRequest.php

```diff
@@ -167,7 +168,13 @@ public function sendRequest($method = 'POST', $uri = self::REQUEST_TYPE_SHIPMENT
         $request->close();
 
         if ($this->getError()) {
-            throw new ApiException('Error in MyParcel API request: ' . $this->getError() . ' Url: ' . $url . ' Request: ' . $this->body);
+            switch ($this->errorCode[0]) {
```

Comment:
```suggestion
            switch (array_key_first($this->errorCode)) {
```

Reply:
of  `switch ($this->errorCode->first())`

Reply:
Kan dat wel op een array? Of is dit een collection?